.github/workflows/reusable-build-system-test-react-native.yml

```diff
@@ -16,24 +16,97 @@ jobs:
       fail-fast: false
       matrix:
         framework: [react-native]
-        framework-version: [latest]
-        # uncomment to re-enable
-        # build-tool: [cli, expo]
-        build-tool: [expo]
+        framework-version:
+          [
+            # uncomment to enable
+            # { formatted: latest, value: latest },
```

Comment:
The existing tests weren't actually using `latest` RN (0.76) hence the need to comment it out here. The existing tests are using latest *Expo* (51), which installs RN 0.74 by default. Expo 52 will support RN `latest`, but it's currently in beta. So this isn't actually reducing our test coverage but we should still try to get 0.76 supported asap